### DISCUSSION TITLE
VAD performance is not good with AAC file types.

### DISCUSSION THREAD

**Discussion by sense2k8** [snakers4/silero-vad#168] [Q&A]
VAD shows very good performance with .wav file. But shows very poor performance with .aac files. Is it some limitations, that silero works only with .wav files only?

**Comment by snakers4** [comment]
Please check that sample rate is handled correctly
It should work with any audio, provided that it is at least in tolerable quality

**Reply by sense2k8** [reply]
> 

Yes sample rate is handled correctly and kept 16000. Is there a way to find the audio quality is in tolerable range or not?

**Reply by snakers4** [reply]
Can you just send your audio?
Does performance change, if you apply any other codec?
